test/unit/use_libtest_harness/use_libtest_harness_test.bzl

```diff
@@ -36,7 +36,10 @@ def _use_libtest_harness_test():
 
     rust_test(
         name = "mytest_noharness",
-        srcs = ["mytest_noharness.rs"],
+        srcs = [
```

Comment:
Is this PR a breaking change? Do test targets with a single source file not using the test harness now require a `main.rs` or will they be able to function just as they did before?